src/wasm32/ffi.c

```diff
@@ -170,7 +170,7 @@ unbox_small_structs, (ffi_type type_ptr), {
 
 EM_JS_MACROS(
 void,
-ffi_call, (ffi_cif * cif, ffi_fp fn, void *rvalue, void **avalue),
+ffi_call_helper, (ffi_cif *cif, ffi_fp fn, void *rvalue, void **avalue),
```

Comment:
Maybe call this `ffi_call_js`?

Reply:
Good point, perhaps that's clearer. I aligned/borrowed this naming convention with those other `EM_JS` functions.

@hoodmane WDYT? Should we use `*_js` here (and possibly those occurrences) instead of `*_helper`?

Reply:
I cherry picked this commit using `ffi_call_js`.

Reply:
I just opened PR https://github.com/hoodmane/libffi-emscripten/pull/22 for this.